### ISSUE TITLE
Lowercase_To_Uppercase

### ISSUE THREAD

**Issue by ujjwalban** [Lakhankumawat/LearnCPP#24]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Lakhankumawat** [comment]
Okay its fine but in these fields mention the issue type here like
>Is your feature request related to a problem? Please describe.
and similar like these

**Comment by Lakhankumawat** [comment]
Describe your issue then it will be assigned to you then you can create a pull request mention this issue number there

**Comment by ujjwalban** [comment]
> r feature request relate

done

**Comment by Lakhankumawat** [comment]
What is the name of algorithm ?

**Comment by Lakhankumawat** [comment]
As far I think there is no specific algorithm for this , it is just a problem statement bro , you can add a problem statement of level 1 regarding this I think

**Comment by Lakhankumawat** [comment]
See here how to contribute an problem statement [Click Here](https://github.com/Lakhankumawat/LearnCPP#problem-statement)

**Comment by ujjwalban** [comment]
yes bhaiya I get it close this issue for now..

**Comment by Lakhankumawat** [comment]
You want to work on this issue ? 
First it will be assigned to you then you will submit a pull request regarding it and then it will be closed .

**Comment by ujjwalban** [comment]
> You want to work on this issue ? First it will be assigned to you then you will submit a pull request regarding it and then it will be closed
yes bhaiya all set. Assign me this issue . I am proper newbie :)

**Comment by Lakhankumawat** [comment]
No worry you can take time and ask me for help .... everyone is newbie in starting so .... contribute as much as you can to practice and get familiar with git and github .
